### PR TITLE
Update real world example in quick start documentation

### LINKED ISSUE #3534: Real World Example Tutorial is not a Real World Example.
https://toil.readthedocs.io/en/latest/gettingStarted/quickStart.html#a-more-real-world-example

This should be changed to an example of running an RNA seq workflow.  ^

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/TOIL-847)
┆Issue Number: TOIL-847

### PR DESCRIPTION
**Pull request by stxue1**
Closes #3534

## Changelog Entry
To be copied to the [draft changelog](https://github.com/DataBiosphere/toil/wiki/Draft-Changelog) by merger:

 * toil-wdl-runner will bump container memory if below 4MiB
 * Updated real world example in quick start
 * `--importWorkersThreshold` has been renamed to `--importWorkersBatchsize`

## Reviewer Checklist

 * [ ] Make sure it is coming from `issues/XXXX-fix-the-thing` in the Toil repo, or from an external repo.
    * [ ] If it is coming from an external repo, make sure to pull it in for CI with:
        ```
        contrib/admin/test-pr otheruser theirbranchname issues/XXXX-fix-the-thing
        ```
    * [ ] If there is no associated issue, [create one](https://github.com/DataBiosphere/toil/issues/new).
* [ ] Read through the code changes. Make sure that it doesn't have:
    * [ ] Addition of trailing whitespace.
    * [ ] New variable or member names in `camelCase` that want to be in `snake_case`.
    * [ ] New functions without [type hints](https://docs.python.org/3/library/typing.html).
    * [ ] New functions or classes without informative docstrings.
    * [ ] Changes to semantics not reflected in the relevant docstrings.
    * [ ] New or changed command line options for Toil workflows that are not reflected in `docs/running/{cliOptions,cwl,wdl}.rst` 
    * [ ] New features without tests.
* [ ] Comment on the lines of code where problems exist with a review comment. You can shift-click the line numbers in the diff to select multiple lines.
* [ ] Finish the review with an overall description of your opinion.

## Merger Checklist

* [ ] Make sure the PR passed tests, including the Gitlab tests, for the most recent commit in its branch.
* [ ] Make sure the PR has been reviewed. If not, review it. If it has been reviewed and any requested changes seem to have been addressed, proceed.
* [ ] Merge with the Github "Squash and merge" feature.
    * [ ] If there are multiple authors' commits, add [Co-authored-by](https://github.blog/2018-01-29-commit-together-with-co-authors/) to give credit to all contributing authors.
* [ ] Copy its recommended changelog entry to the [Draft Changelog](https://github.com/DataBiosphere/toil/wiki/Draft-Changelog).
* [ ] Append the issue number in parentheses to the changelog entry.

### CODE REVIEW

**Comment by stxue1** [comment]
Also closes #5316

File: docs/wdl/running.rst [hunk 1]
```diff
@@ -84,6 +86,22 @@ the `Cromwell Output Organizer (croo)`_, will default to ``True``.
...
+
+``--runImportsOnWorkers``: Run file imports on workers. This is useful if the leader is not network optimized
+and lots of downloads are necessary. By default, this is false.
+
+``--importWorkersThreshold``: Requires ``--runImportsOnWorkers`` to be true. Specify the target batch size in bytes for batched imports.
+As many files as can fit will go into each batch import job. This also accepts abbreviations, such as ``G`` or ``Gi``.
+
+``--importWorkersDisk``: Requires ``--runImportsOnWorkers`` to be true. Specify the disk size each import worker will get.
+This may be necessary when file streaming is not possible. For example, downloading from AWS to a GCE job store.
+If specified, this should be set to the largest file size of all files to import. By default, this is 1 MiB.
```
Comment (adamnovak) [inline docs/wdl/running.rst:103]: Usually the user won't have to set this at all, because Toil will be able to read the file size from the source and allocate enough disk. The user only has to touch this when that isn't possible. The documentation should reflect that and should be about increasing the disk allocation for downloads that can't work otherwise.

Comment (adamnovak) [reply docs/wdl/running.rst]: Hm, probably we should make *some* attempt to allocate the right amount of disk at some point.

**Review by adamnovak — APPROVED** [review]
I think this is good now.

File: docs/wdl/running.rst (hunk 1 above)
Comment (stxue1) [reply docs/wdl/running.rst]: I think the issue then is having some way to figure out if file streaming is supported or not. I had a previous implementation that would attempt to download on a worker with almost no disk space, then catch a crash and allocate the right amount of disk space, but I think we removed it since it was too hacky. This may also still be needed as an argument if we can't sniff file size out of servers.

**Comment by adamnovak** [comment]
We might actually want to show a copy of the error we get without the `--quantCheck=False` flag. Because the error doesn't suggest to use the flag (maybe we need to actually parse the errors and suggest the flag), so we'd want the documentation to come up in search when people search the error.

File: src/toil/test/wdl/wdltoil_test.py [hunk 2]
```diff
@@ -1121,122 +1120,7 @@ def make_string_expr(self, to_parse: str) -> WDL.Expr.String:
         for i in range(1, len(parts), 2):
             parts[i] = WDL.Expr.Placeholder(pos, {}, WDL.Expr.Null(pos))
 
-        return WDL.Expr.String(pos, parts)
-
-    def test_remove_common_leading_whitespace(self) -> None:
```
Comment (adamnovak) [inline src/toil/test/wdl/wdltoil_test.py:1126]: Since we're removing these test that exercise a lot of edge cases, are we sure we have enough tests for the overall behavior (i.e. in workflows in the conformance tests or in Toil) that we'll catch breakage if MiniWDL changes its behavior and stops following the spec again?

**Review by adamnovak — APPROVED** [review]
I think this will work. I am a little worried removing all the edge case whitespace tests opens us up to breakage if MiniWDL changes again, but it might not be worth porting them all to e.g. conformance tests.
